imports/ui/App.js

```diff
@@ -22,11 +26,20 @@ export default class App extends React.Component {
 
     render() {
         return (
-            <div>
-                <AppBar title="Meteor React Latest" onLeftIconButtonTouchTap={this.toggleMenu} />
-                <Menu open={this.state.open} onSelectMenu={this.closeMenu} />
-                <Routes/>
-            </div>
+            [
+                <AppBar key="appBar">
+                    <Toolbar>
+                        <IconButton color="contrast" aria-label="Menu" onClick={this.toggleMenu}>
+                            <MenuIcon />
+                        </IconButton>
+                        <Typography type="headline" color="inherit">
+                            Meteor React Latest
+                        </Typography>
+                    </Toolbar>
+                </AppBar>,
+                <Menu key="menu" open={this.state.open} onSelectMenu={this.closeMenu} />,
+                <main key="routes" style={{marginTop:60}}><Routes/></main>
```

Comment:
What is main? Why not div or nothing?

Reply:
I have chosen the main tag to specify this place as a main content of the page, but it can also be a div tag. I need something to add marginTop style, Routes component not support this.

Reply:
I don't see a lot of people using main, etc, I guess that is better to use div :)

Reply:
Done.